h/static/scripts/streamer.js

```diff
@@ -27,32 +21,15 @@ var socket;
  * @return The push notification service client.
  */
 // @ngInject
-function connect($rootScope, annotationMapper, groups, session, settings) {
-  // The public interface of the streamer. Returned from connect.
-  var controls = {
-    setConfig: setConfig
-  };
+function Streamer($rootScope, annotationMapper, groups, session, settings) {
+  // The randomly generated session UUID
+  var clientId = uuid.v4();
+
   // Client configuration messages, to be sent each time a new connection is
   // established.
   var configMessages = {};
 
-  // Close any existing socket
-  if (socket) {
-    socket.close();
-  }
-
-  // If we have no URL configured, don't do anything.
```

Comment:
Previously this gracefully degraded if `websocketUrl` wasn't provided in the settings, and fell back to not connecting. It looks to me like that's no longer true.